insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
```

Comment:
This is interesting, but it never actually explains why this is of any relevance at all.

Why do we even need this?  How would it be used?  What problems does having a user or group name starting with a digit cause?

If there is a reason - and I know there is - then please add it.

In general I'm not hugely keen on having parsers that check for a complicated condition that's only useful to a single rule.  I understand the reason, but they add to the processing time and load on the client.